### ISSUE TITLE
Improve map rotation/zoom behaviour

### ISSUE THREAD

**Issue by cbalster** [criticalmaps/criticalmaps-android#249]
Currently when pinching/rotating the map it's not really possible to only zoom without also rotating the map. This does not feel ideal. Also a user reported that this might be problematic for people with dizziness.
Ideally the behavior would be similar to google maps, which seems to - within the first few pixels of touch - distinguishes between rotate and zoom and ignores rotation within a certain window while zooming.
Unfortunately doing this is not that simple due to the way gesture detection is implemented in osmdroid and might require quite a lot of work and should probably done directly in the osmdroid codebase.
Until then I see 2 options:
1) find an easier workaround we can implement directly
2) until then add an option to disable map rotation completely

**Comment by zutrinken** [comment]
Agree. We could also disable rotation for the moment completely without an extra setting. You can still rotate your device 🙃

**Comment by cbalster** [comment]
😁
I don't think the people that requested it before would be happy about
that.

**Comment by dreua** [comment]
I'd also like single handed zoom gestures like in Google Maps and maps.me: 
Tap + Drag up: Zoom Out
Tap + Drag down: Zoom In
